packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -90,21 +99,94 @@ const inlineMobileStyles = ({
     ${getInlineStyles({ theme })}
   `;
 
+const actionWrapperStyles = ({ theme }: StyleProps) => css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+
+  ${theme.mq.kilo} {
+    flex-direction: row;
+  }
+`;
+
+const buttonStyles = ({ theme }: StyleProps) => css`
+  width: 100%;
+
+  ${theme.mq.kilo} {
+    width: auto;
+  }
+`;
+
+const PrimaryButton = styled(Button)(buttonStyles);
+
+const secondaryButtonStyles = ({ theme }: StyleProps) => css`
+  margin-right: ${theme.spacings.mega};
+  ${theme.mq.untilKilo} {
+    display: none;
+  }
+`;
+
+const SecondaryButton = styled(Button)(secondaryButtonStyles, buttonStyles);
+
+const tertiaryButtonStyles = ({ theme }: StyleProps) => css`
+  margin-top: ${theme.spacings.mega};
+  ${theme.mq.kilo} {
+    display: none;
+  }
+`;
+
+const TertiaryButton = styled(Button)(tertiaryButtonStyles, buttonStyles);
+
 const Wrapper = styled('div')<ButtonGroupProps>(
   baseStyles,
   alignmentStyles,
   inlineMobileStyles,
 );
 
+const ActionsWrapper = styled('div')<ButtonGroupProps>(actionWrapperStyles);
+
 /**
  * Groups its Button children into a list and adds margins between.
  */
 export const ButtonGroup = forwardRef(
-  ({ children, ...props }: ButtonGroupProps, ref: ButtonGroupProps['ref']) => (
-    <Wrapper {...props} ref={ref}>
-      {children}
-    </Wrapper>
-  ),
+  (
+    { children, action, ...props }: ButtonGroupProps,
+    ref: ButtonGroupProps['ref'],
+  ) => {
+    if (action) {
+      return (
+        <ActionsWrapper {...props}>
+          <SecondaryButton
+            variant="secondary"
+            onClick={action.secondary.onClick}
+            href={action.secondary.href}
+          >
+            {action.secondary.children}
+          </SecondaryButton>
+          <PrimaryButton
+            variant="primary"
+            onClick={action.primary.onClick}
+            href={action.primary.href}
+          >
+            {action.primary.children}
+          </PrimaryButton>
+          <TertiaryButton
+            variant="tertiary"
+            onClick={action.secondary.onClick}
+            href={action.secondary.href}
+          >
+            {action.secondary.children}
+          </TertiaryButton>
```

Comment:
You need to spread the props on each Button:
```suggestion
          {action.secondary && (
          	<SecondaryButton
          	  {...action.secondary}
           	 variant="secondary"
          	/>
          )}
          <PrimaryButton
          	{...action.primary}
            variant="primary"
          />
          {action.secondary && (
	        <TertiaryButton
		      {...action.secondary}
	          variant="tertiary"
	        />
	      )}
```